1.0-rc.1.md

```diff
@@ -0,0 +1,101 @@
+
+- It is not possible to upgrade directly to this version of CockroachDB from a version older than `beta-20170413`. Upgrade any older clusters to that version first. [#15228](https://github.com/cockroachdb/cockroach/pull/15228)
+- The `cockroach start` command no longer recognizes the `COCKROACH_INSECURE` and `COCKROACH_CERTS_DIR` environment variables which are intended to be client-only. Use the `--insecure` and `--certs-dir` command-line flags instead. [#15196](https://github.com/cockroachdb/cockroach/pull/15196) [#15288](https://github.com/cockroachdb/cockroach/pull/15288)
+- Older certificate-related command-line flags have been removed; use `--certs-dir` instead. [#15241](https://github.com/cockroachdb/cockroach/pull/15241)
+- Various environment variables have been removed in favor of the new `CLUSTER SETTING` commands described below.
+- The undocumented `SET SYNTAX` statement and the "modern" syntax variant have been removed. [#15064](https://github.com/cockroachdb/cockroach/pull/15064)
+
+### General Changes
+
+- Anonymous usage statistics are now reported by default. To opt out, set `COCKROACH_SKIP_ENABLING_DIAGNOSTIC_REPORTING=true` in the environment when starting the server or use `SET CLUSTER SETTING diagnostics.reporting.enabled = false` in the SQL shell for a running cluster. [#14901](https://github.com/cockroachdb/cockroach/pull/14901)
+
+### Build Changes
+
+- CockroachDB now builds on Windows. Windows support is currently experimental and is recommended only for development, not production.
+- Building from source on macOS 10.11 works again. [#15168](https://github.com/cockroachdb/cockroach/pull/15168)
+- Building from source on older CPUs (without SSE4.2) works again. [#15247](https://github.com/cockroachdb/cockroach/pull/15247)
+
+### SQL Language Changes
+
+- New commands `SET CLUSTER SETTING` and `SHOW ALL CLUSTER SETTINGS` can now be used to change certain cluster-wide settings. The following cluster settings are available in this release:
+  - `diagnostics.reporting.enabled`
+  - `enterprise.enabled`
+  - `kv.raft.command.max_size`
+  - `kv.raft_log.synchronize`
+  - `kv.snapshot_rebalance.max_rate`
+  - `kv.snapshot_recovery.max_rate`
+  - `kv.transaction.max_intents`
+  - `server.remote_debugging.mode`
+  - `sql.metrics.statement_details.dump_to_logs`
+  - `sql.metrics.statement_details.enabled`
+  - `sql.metrics.statement_details.threshold`
+  - `sql.trace.log_statement_execute`
+  - `sql.trace.session_eventlog.enabled`
+  - `sql.trace.txn.enable_threshold`
+- The `PREPARE` and `EXECUTE` statements can now be used to use prepared statements in a SQL shell. [#15256](https://github.com/cockroachdb/cockroach/pull/15256)
+- The `regexp_replace` function now handles newlines correctly, both with and without the `n` flag. [#15309](https://github.com/cockroachdb/cockroach/pull/15309)
+- Precedence for the binary `~` operator (regex match) has been changed. [#15264](https://github.com/cockroachdb/cockroach/pull/15264)
```

Comment:
May need to add: `This could cause DEFAULT or CHECK expressions to have different behavior if ~ was used in combination with other operators. These will need to be fixed manually.`